examples/SimpleDrone/SimpleDrone.py

```diff
@@ -0,0 +1,244 @@
+
```

Comment:
@tcr3dr The fact that people are writing this sort of thing tells me we still haven't got the interface for DKPY right. From this we need better/clearer access to the geofence, and also velocity control in the earth frame (whatever that is!)